src/algebra/group_with_zero/basic.lean

```diff
@@ -1032,15 +1032,15 @@ section group_with_zero
 variables (f : monoid_with_zero_hom G₀ G₀') (a b : G₀)
 
 /-- A monoid homomorphism between groups with zeros sending `0` to `0` sends `a⁻¹` to `(f a)⁻¹`. -/
-@[simp] lemma map_inv' : f a⁻¹ = (f a)⁻¹ :=
```

Comment:
Maybe this should just be called `f.map_inv`, since `map_div` isn't primed below.

Reply:
Agreed. I'll heading off now, but I'll leave CI to check if it worked.